plugin-tester-scala/src/main/scala/example/myapp/helloworld/GreeterClient.scala

```diff
@@ -22,24 +22,27 @@ import scala.util.{ Failure, Success }
 object GreeterClient {
 
   def main(args: Array[String]): Unit = {
-
+    // Boot akka
```

Comment:
We need Akka-Boot :tm:

Reply:
:boot: Akka Boots :boot:

Reply:
https://www.youtube.com/watch?v=zuDtACzKGRs ?

Reply:
:beer: :boot: